### DISCUSSION TITLE
Module Monikers

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1640] [Q&A]
DNN has something called Module Monikers - "A unique identifier for this module on the page. This can be used by the WEB API callers to set the context [sic] of the Tab and Module context [sic]." (I wonder if "context" should be "content" in this DNN tooltip.)

At the very least, I can see where just being able to give a custom name to a section, or even a block, could be useful.

**Comment by meetmandeep** [comment]
This is already possible. Use the "Navigator", double click on a block to rename. 

See: 
![image](https://user-images.githubusercontent.com/29958123/130105134-309191d8-b47f-4755-be44-584a90b6d55d.png)

It would be great to edit the name from the block itself. Feel free to add that as a feature request in Issues.
